docs/api/resolutions.rst

```diff
@@ -63,13 +63,15 @@ HasLength
 **Aliases**: ``HaveLength``
 
 .. autoclass:: HasLength
+    :exclude-members: length_to_log, item_plural
 
 IsCloseTo
 ---------
 
 **Aliases**: ``CloseTo``
 
 .. autoclass:: IsCloseTo
+    :exclude-members: delta_to_log, num_to_log
```

Comment:
As mentioned in [the comments below](https://github.com/ScreenPyHQ/screenpy/pull/141/files#r1612025535), you can remove these `:exclude-members:` directives because we don't have `:members:` on here to begin with.

Reply:
Gotcha.  Removing them.  (I really ought to take some time to learn more about autodoc)